dipu/torch_dipu/csrc_dipu/aten/ops/AutoCompareUtils.hpp

```diff
@@ -192,5 +192,19 @@ inline std::string allclose_autocompare(
   return stream.str();
 }
 
+
+template<typename T, std::enable_if_t<std::is_arithmetic<T>::value, bool> = true>
+inline std::string allclose_autocompare(T val_expect,  T val_real) {
```

Comment:
```suggestion
template<typename T, std::enable_if_t<std::is_arithmetic_v<T>, bool> = true>
std::string allclose_autocompare(T val_expect,  T val_real) {
```
该重载少一个参数，上层有没有泛型会出问题？

Reply:
这里不会，但是少一个那个关于缩进的参数确实不好，我改下。